src/libs/localStore/storage.ts

```diff
@@ -27,7 +27,10 @@ export const storeData = async (
   value: StorageItems[keyof StorageItems]
 ): Promise<void> => {
   try {
-    await setItem(name, JSON.stringify(value))
+    // Convert value to a string before storing it
+    const stringValue =
+      typeof value === 'boolean' ? String(value) : JSON.stringify(value)
```

Comment:
What is the benefit to do that? `JSON.stringify` already convert boolean to string 🤔

Reply:
It was tied to an issue I had during testing with the AsyncStorage mock. I should look into it again

- [ ] Investigate